pkg/apis/v1alpha5/provisioner.go

```diff
@@ -49,7 +51,7 @@ type ProvisionerSpec struct {
 	// +optional
 	StartupTaints []v1.Taint `json:"startupTaints,omitempty"`
```

Comment:
Don't we need to hash this too?

Reply:
Yes, startup taints are included in the hash. All fields that don't include `hash:"ignore"` will be hashed